integration-cli/docker_cli_plugins_test.go

```diff
@@ -26,7 +30,15 @@ func (s *DockerSuite) TestPluginBasicOps(c *check.C) {
 
 	out, _, err = dockerCmdWithError("plugin", "inspect", pNameWithTag)
 	c.Assert(err, checker.IsNil)
-	c.Assert(out, checker.Contains, "A test plugin for Docker")
+	tmpFile, err := ioutil.TempFile("", "inspect.json")
+	c.Assert(err, checker.IsNil)
+	defer tmpFile.Close()
+
+	if _, err := tmpFile.Write([]byte(out)); err != nil {
+		c.Fatal(err)
+	}
+	id, err := exec.Command("jq", ".Id", "--raw-output", tmpFile.Name()).CombinedOutput()
```

Comment:
@anusha-ragunathan can we add a comment here (a FIXME), to remember to update when `plugin inspect` will take a format ? 👼